### ISSUE TITLE
The inspect_result has NaNs/zero where it should not: check data

### ISSUE THREAD

**Issue by fepegar** [thenineteen/Semiology-Visualisation-Tool#53]
When the query is, for example, "Figure of 4".

**Comment by thenineteen** [comment]
does it also give this error for Myoclonic or Other Simple Motor?

**Comment by fepegar** [comment]
You can try yourself with a script like this:

```python
import mega_analysis
from mega_analysis import Semiology, Laterality
semiology = Semiology(
    'Figure of 4',
    symptoms_side=Laterality.LEFT,
    dominant_hemisphere=Laterality.LEFT,
)
num_patients_dict = semiology.get_num_patients_dict()
print('Result:', num_patients_dict)
```

**Comment by fepegar** [comment]
This was closed automatically because you added "Fix #53" in the commit title.

Also, please don't reference issues in commit titles but in commit messages.
